contrib/utilities/indent

```diff
@@ -35,35 +35,36 @@ if test ! -d source -o ! -d include -o ! -d examples ; then
   exit 1
 fi
 
-# Add the location 'setup_astyle.sh' installs astyle to to the local PATH.
-ASTYLE_PATH="$(cd "$(dirname "$0")" && pwd)/programs/astyle/bin"
-export PATH=$ASTYLE_PATH:$PATH
+# Add the location 'download_clang_format' or 'compile_clang_format'
+# installs clang-format to to the local PATH.
+CLANG_FORMAT_PATH="$(cd "$(dirname "$0")" && pwd)/programs/clang-6/bin"
+export PATH=$CLANG_FORMAT_PATH:$PATH
 
-if test ! -f contrib/styles/astyle.rc ; then
-  echo "*** No style file contrib/styles/astyle.rc found."
-  exit 1
-fi
-
-if test -z "$(command -v astyle)" ; then
-  echo "*** No astyle program found."
+if ! [ -x "$(command -v clang-format)" ]; then
+  echo "*** No clang-format program found."
   echo "***"
-  echo "*** You can download astyle from http://astyle.sourceforge.net/"
-  echo "*** Note that you will need exactly version 2.04 (no newer or"
-  echo "*** older version will yield the correct indentation)."
-  echo "*** You can run the 'setup_astyle.sh' script for that."
+  echo "*** You can run the 'download_clang_format' script or the"
+  echo "*** 'compile_clang_format' script for obtaining a compatible"
+  echo "*** binary and installing to an appropriate directory."
   exit 1
 fi
 
-if test "$(astyle --version 2>&1)" != "Artistic Style Version 2.04" ; then
-  echo "*** Found a version of astyle different than the required version 2.04."
+# Make sure to have the right version. We know that clang-6.0.0
+# and clang-6.0.1 work. Hence, test for clang-6.0.
+CLANG_FORMAT_VERSION=$(clang-format --version)
```

Comment:
Please quote: `CLANG_FORMAT_VERSION="$(clang-format --version)"` (just for the looks).